pkg/controller/cloud/node_controller.go

```diff
@@ -367,6 +367,8 @@ func (cnc *CloudNodeController) AddCloudNode(obj interface{}) {
 		utilruntime.HandleError(err)
 		return
 	}
+
+	glog.Infof("Successfully initialized node %s with cloud provider", node.Name)
```

Comment:
What v-level is this by default? Perhaps be consistent with `V(2)` here?

Reply:
I believe it's 0 by default, so it will always log. This was intentional :)

Reply:
Ok, great!